content/300-guides/200-deployment/110-deployment-guides/500-deploying-to-netlify.mdx

```diff
@@ -79,75 +72,95 @@ npm install
 ```terminal
 ls -1
 README.md
-functions
+functions/
 netlify.toml
-node_modules
+node_modules/
 package.json
-prisma
-public
-schema.sql
+prisma/
+public/
 ```
 
-## 3. Set the DATABASE_URL environment variable locally
+## 3. Get the database connection string
 
-Set the `DATABASE_URL` environment variable locally so you can create the database schema and Prisma can access the database to introspect:
+You need the database connection string to define the `DATABASE_URL` environment variable.
 
-```terminal
-export DATABASE_URL="postgresql://__USER__:__PASSWORD__@__HOST__/__DATABASE__"
-```
+If you are using Heroku, follow the steps below.
 
-> **Note:** you will need the `DATABASE_URL` environment variable for the subsequent steps. Set it in all terminal sessions related to this project.
+1. In Heroku, open your app and select the **Resources** tab.
+2. Click the **Heroku Postgres** database add-on.
+3. Select the **Settings** tab in the page of the Heroku Postgres add-on.
+4. Click **View Credentials**.
+5. From the **URI** field, copy the entire connection string.
 
-You need to replace the uppercase placeholders with your database credentials, e.g.:
+**Checkpoint:** The database connection string has the following format:
 
-```terminal
+```
+postgresql://__USER__:__PASSWORD__@__HOST__/__DATABASE__
+```
+
+For example:
+
+```
 postgresql://janedoe:randompassword@yourpostgres.compute-1.amazonaws.com:5432/yourdbname
 ```
 
-## 4. Create the database schema
+## 4. Set the <inlinecode>DATABASE_URL</inlinecode> variable in a <inlinecode>.env</inlinecode> file
+
+You can store local environment variables related to Prisma in an `.env` file. For example, instead of setting the database connection string as the local environment variable `DATABASE_URL`, you can keep it in an `.env` file. When you run `prisma db push` later, Prisma automatically looks for any variables defined in `.env`.
```

Comment:
> When you run `prisma db push` later, Prisma automatically looks for any variables defined in `.env`.

This is _technically_ not correct. Prisma CLI will look at `.env` and load them into the environment via the `dotenv` library. Later `prisma db push` will just read what is in the environment (which might be the values from `.env`) - so a bit nitpicky.